app/extensions/brave/locales/en-US/bravery.properties

```diff
@@ -25,7 +25,12 @@ httpsEverywhere=HTTPS Everywhere
 noScript=Block Scripts
 noScriptPref=Block Scripts (will break many sites)
 safeBrowsing=Block Phishing / Malware
+safeBrowsingControl=Block Phishing / Malware *
+advancedSafeBrowsing=Advanced Safe Browsing
+advancedSafeBrowsingInfo=* Advanced safe browsing supplements Brave's phishing and malware protection with Google Safe Browsing.
```

Comment:
cc @flamsmark for thoughts on wording

Reply:
This wording is okay, but I don't think it really teases out the differences.

`````
+safeBrowsingControl=Block Phishing and Malware *
+advancedSafeBrowsing=Protection plus Google Safe Browsing 
+advancedSafeBrowsingInfo=* You can supplement Brave's phishing and malware protection with Google's Safe Browsing service.
+basicSafeBrowsing=Phishing and malware protection
+disableSafeBrowsing=No phishing and malware protection
`````